.pre-commit-config.yaml

```diff
@@ -37,11 +37,12 @@ repos:
           - --fix
           - --show-fixes
       - id: ruff-format
-  # # type checking
-  # - repo: https://github.com/pre-commit/mirrors-mypy
-  #   rev: v1.17.0
-  #   hooks:
-  #     - id: mypy
+  # type checking
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.18.1
+    hooks:
+      - id: mypy
+        exclude: 'tests/.*'
```

Comment:
Does checking the test files cause failures?

Reply:
Note that something like spacetelescope/rad#698's suggestion would solve the issue as mypy will account for them based off the stub file.